Alexandr_hw_4.js

```diff
@@ -0,0 +1,165 @@
+// ## Complete exercise
+
+// ### NORMAL level
+
+// #### Task 1 💻
+
+// function getSum (con){
+//     let sum = 0;
+//     for (let i = 0; i <= con; i++){
+//         sum = sum + i;
+//     }
+//     return sum
+// }
+// console.log(getSum(100));
+
+// #### Task 2 💻
+
+// const PERSENT = 17;
+// const AGE = 5;
```

Comment:
use Uppercase only for real constants like a const pi = 3.14. But for simple variable it should be const age = 5;